### DISCUSSION TITLE
Can I set inView true?

### DISCUSSION THREAD

**Discussion by aditi-gawli** [thebuilder/react-intersection-observer#639] [Q&A] [answered]
I want to scroll to a particular element in DOM when user clicks a button, can this be achieved by setting inView true for that element id ?

**Comment by thebuilder** [comment, marked as answer]
No. The IntersectionObserver is just a reporter.

You could use something like [scrollIntView](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView), or simply a `#hash` link.
